charts/ingress-nginx/values.yaml

```diff
@@ -527,7 +527,8 @@ controller:
       # -- Annotations are mandatory for the load balancer to come up. Varies with the cloud service.
       annotations: {}
 
-      # loadBalancerIP: ""
+      # -- Used by cloud providers to connect the resulting internal LoadBalancer to a pre-existing static IP. Make sure to add to the service the needed annotation to specify the subnet which the static IP belongs to. For instance, `networking.gke.io/internal-load-balancer-subnet` for GCP and `service.beta.kubernetes.io/aws-load-balancer-subnets` for AWS.
+      loadBalancerIP: ""
```

Comment:
Please comment out the annotation like it is in the original

Reply:
Hi @strongjz , I didn't get the reason. The variable `loadBalancerIP` is correctly handled into the file [controller-service-internal.yaml](https://github.com/kubernetes/ingress-nginx/blob/24b8e32b088bec15e1f40deab8fefc8bd1b6c2da/charts/ingress-nginx/templates/controller-service-internal.yaml) but it's not present into the documentation.
The aim of this PR is to make visible the helm value `controller.service.internal.loadBalancerIP` into the helm documentation regarding the internal load balancer, at the same way the helm value `controller.service.loadBalancerIP` is already documented for deploying an external load-balancer [controller-service.yaml](https://github.com/kubernetes/ingress-nginx/blob/24b8e32b088bec15e1f40deab8fefc8bd1b6c2da/charts/ingress-nginx/templates/controller-service.yaml)
I read your previous message in which you listed the yaml files to deploy specifically for cloud providers. Those files are good but they don't deal with Helm. They are suitable for who has adopted the kubectl strategy.

Please could you elaborate a little bit your POV? And can you explain why `controller.service.loadBalancerIP` is not commented and `controller.service.internal.loadBalancerIP` should be commented in your view? Thank you.

Reply:
It's empty by default, so it's fine. I needed to understand more of the helm flow control.